src/main/resources/assets/modmanager/lang/ru_ru.json

```
@@ -9,6 +9,9 @@
     "modmanager.button.defaultProvider": "Источник по умолчанию",
     "modmanager.button.updateChannel": "Канал обновления",
     "modmanager.button.save": "Сохранить",
+    "modmanager.button.hide": "Скрыть обновления",
+    "modmanager.button.show": "Показать обновления",
+    "modmanager.button.open": "Открыть ModManager",
```

Should I translate the mod name to match the logical meaning of the button?
```suggestion
    "modmanager.button.open": "Открыть менеджер модов",
```

---

yeah the logical meaning is better than a 1:1 translation